assets/js/renderer.js

```diff
@@ -388,8 +388,31 @@ $(document).ready(function () {
     // Center and reset node view
     resetZoomButton.click(() => nodeView.centerNode());
 
+    // Instantiate the splitter.
+    const splitter = Split(['#leftCol', '#tabbar'], {
+        sizes: [18, 76],
+        minSize: 200,
+        gutterSize: 5,
+        onDragEnd: function() {
+            sessionStorage.setItem('splitSizes', JSON.stringify(splitter.getSizes()));
+        }
+    });
+
+    // Retrieve any saved resize settings from sessionStorage or else use our default setting.
+    function restoreSplitterSizes() {
+        let splitSizes;
```

Comment:
Nice catch @jdaguil  -- thank you!  This solution seems far better than one of the alternative approaches we discussed yesterday (handling the `splitSizes` variable in the `catch` statement).  I'll tackle as soon as I get in this morning.